monkey/infection_monkey/network/info.py

```diff
@@ -21,6 +22,41 @@
 RTF_REJECT = 0x0200
 
 
+class IntRange:
```

Comment:
Should this go in `common.types`?

Reply:
I don't mind moving it. Currently it is only used here, but it may be useful elsewhere

Reply:
I like defining things close to where they're used. But this does seem very generic.